src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1014,19 +1015,44 @@ public function insert_posts( $items = array() ) {
 				}
 			}
 
-			//if we are setting all events to a category specified in saved import
+			// if we are setting all events to a category specified in saved import
 			if ( ! empty( $this->meta['category'] ) ) {
 				$terms[] = (int) $this->meta['category'];
 			}
 
 			wp_set_object_terms( $event['ID'], $terms, Tribe__Events__Main::TAXONOMY, false );
 
+			// If we have a Image Field from Service
 			if ( ! empty( $event['image'] ) ) {
-				$attachment = $this->import_event_image( $event['ID'], $event );
+				// Attempt to grab the event image
+				$image_import = Tribe__Events__Aggregator::instance()->api( 'image' )->get( $event['image']->id );
 
-				if ( $attachment ) {
-					// Log this attachment was created
-					$activity->add( 'attachment', 'created', $attachment );
+				/**
+				 * Filters the returned event image url
+				 *
+				 * @param array|bool $image       Attachment information
+				 * @param array      $event       Event array
+				 */
+				$image = apply_filters( 'tribe_aggregator_event_image', $image_import, $event );
+
+				// If there was a problem bail out
+				if ( false === $image ) {
+					continue;
+				}
+
+				// Verify for more Complex Errors
+				if ( is_wp_error( $image ) ) {
+					continue;
+				}
+
+				if ( isset( $image->status ) && 'created' === $image->status ) {
```

Comment:
This could be safe - but we had similar code elsewhere and it was possible that the response (`$image` here) was not a WP error but also didn't contain the expected fields.

So, maybe add a check to ensure status exists unless you're positive that's not a risk?